code/modules/events/spider_migration_yw.dm

```diff
@@ -11,9 +11,9 @@
 /datum/event/spider_migration/announce()
 	var/announcement = ""
 	if(severity == EVENT_LEVEL_MAJOR)
-		announcement = "Massive breach of [station_name()] walls detected. Unknown biological entities has been detected near within perimeter. Heavily armed Security Patrol required."
+		announcement = "Massive breach of [station_name()] walls detected. Unknown biological entities has been detected near within walls perimeter. Heavily armed Security Patrol required."
```

Comment:
should probably be something like "...walls detected. Unknown biological entities have been detected within outpost perimeter."

Reply:
You're right. Thats what I get for looking at it at 2 am.